vnext/src/RNTester/js/examples/Accessibility/AccessibilityInfoExample.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) Microsoft Corporation.
```

Comment:
There's currently an allow list of RNTester pages that E2ETest visits. Could you update that with this one? It's in a file like "visitAllPages.ts" or something like that.